### PR TITLE
chore(pipelines): migrate approve lambda handler

### PR DESCRIPTION
**Pull request by colifran**
This PR moves the approve lambda handler from aws-cdk-lib to our new centralized location for custom resource handlers in the [@aws-cdk](https://github.com/aws-cdk) package.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by vinayak-kukreja** [comment]
@Mergifyio update

File: yarn.lock [hunk 1]
```diff
@@ -387,6 +387,52 @@
     tslib "^2.5.0"
     uuid "^8.3.2"
 
+"@aws-sdk/client-codepipeline@^3.445.0":
```
Comment (vinayak-kukreja) [inline yarn.lock:390]: I think this just did not get updated to the fixed version.

Comment (colifran) [reply yarn.lock]: Updated!

**Review by vinayak-kukreja — APPROVED** [review]

**Comment by vinayak-kukreja** [comment]
@Mergifyio update

**Pull request merged**